test/zip-in-zip-test.js

```diff
@@ -0,0 +1,255 @@
+var yauzl = require("../");
+var PassThrough = require("stream").PassThrough;
+var util = require("util");
+var Pend = require("pend");
+var BufferList = require("bl");
+
+exports.runTest = runTest;
+
+// zipfile obtained via:
```

Comment:
thanks for being conscientious enough to add tests :+1: . However, I think we can test this functionality in a much simpler way.

Reply:
Fair enough.  I wanted to sure for certain that ranges within what is basically another range (the zip in the zip) worked properly.